### DISCUSSION TITLE
issue with sort-keys

### DISCUSSION THREAD

**Discussion by brewster1134** [eslint/eslint#16965] [Q&A]
if i have a list like

```
A: 'a',
A_B: 'ab'
```

when it is auto sorted, it switches the order

```
A_B: 'ab'
A: 'a',
```

i want to keep the shorter version first, with the following with additional granularity after, im sure this has been discussed before, but was unable to find anything. i have played around with various `eslintrc` settings, but no luck

i get `Expected object keys to be in natural insensitive ascending order` error

**Comment by Rec0iL99** [comment]
Hi @brewster1134, please explain how it is auto-sorted. 

ESLint will not auto-fix/sort a `sort-keys` rule when reported, it's the responsibility of the user to fix it manually.

I guess this is your rule config:
```js
/* eslint sort-keys: ["error", "asc", {"natural": true, "caseSensitive": false}] */
```
